extensions/adobe/experience/decisioning/option-detail.example.4.json

```diff
@@ -1,6 +1,16 @@
 {
   "xdm:id": "xcore:personalized-offer:e91ee850a0bb7d9",
-  "xdm:selectionID": "xcore:fb30c2f0a6a98e030dd107f2c531c5fe374ce410bb96b02b:184a0fc14872e39c",
-  "xdm:selectionType": "manual",
-  "xdm:trackingToken": "Tracking Token Value"
+  "xdm:score": 101,
+  "xdm:itemSelection": {
+    "xdm:selectionDetail": {
+      "xdm:strategyID": "xcore:selection-strategy:1863a7b4254b8778",
+      "xdm:strategyName": "sample strategy name",
+      "xdm:type": "selectionStrategy",
+      "xdm:version": "xcore:selection-strategy:1863a7b4254b8778_1"
+    },
+    "xdm:rankingDetail": {
+      "xdm:id": "xcore:ranking-function:17bb67e976902620",
+      "xdm:type": "rankingFormula"
```

Comment:
Where is this field coming from?

Reply:
sorry, my mistake. it has been updated to xdm:trafficType.